main_routers/system_router.py

```
@@ -1649,6 +1732,29 @@ async def _llm_call_with_retry(
             except Exception as e:
                 logger.warning(f"[{lanlan_name}] Phase 1 Web 筛选异常: {type(e).__name__}: {e}")
         
+        # 音乐模式特殊处理：不经过 Phase 1 LLM 筛选，直接添加音乐话题
+        if music_content and music_content.get('formatted_content'):
+            music_topic = music_content['formatted_content']
+            if music_topic:
+                # 检查音乐话题是否重复（基于歌曲名去重）
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)